kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.stream.array;
+
+import java.util.OptionalDouble;
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+    static double getAverage(int[] numbers){
+
+        IntStream.range(0,numbers.length)
+                .map(n -> numbers[n])
+                .forEach(n -> System.out.print(n + " "));
+
+        OptionalDouble average = IntStream.range(0,numbers.length)
+                .map(n -> numbers[n])
+                .average();
+                return average.getAsDouble();
```

Comment:
Jeśli nie używasz później nigdzie zmiennej `average` to dobra praktyką jest odrazu jej zwrócenie. 
Tzn : 
return `InsStream.range()......average().getAsDouble()`